### PR TITLE
For CDD Filled Forms , we now use data_verification field to indicate status on Filled Forms

### LINKED ISSUE #1289: Update status on the filled form view for the CDD daily summary form to verified or not verified
Related JIRA ticket https://smartregister.atlassian.net/browse/RVL-1432?focusedCommentId=14119

Instead of returning if task is complete or not under Status on the filled form view for the CDD daily summary form  , return `verified` when answer = `yes`, or `not verified` when answer = `no` for the question "**Has this data been verified with hard copy forms**".

Add Data Collection Date column to the Filled Forms, to display data when the data was collected, versus the date the event was created.

### PR DESCRIPTION
**Pull request by lindagcaba**
resolves #1289

### CODE REVIEW

File: opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json [hunk 1]
```diff
@@ -480,11 +480,11 @@
         "label": "Has this data been verified with hard copy forms",
         "options": [
           {
-            "key": "Yes",
+            "key": "verified",
             "text": "Yes"
```
Comment (Rkareko) [inline opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json:484]: The key and the test should be similar. The way it currently is leads to confusion

File: opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json [hunk 2]
```diff
@@ -480,11 +480,11 @@
...
           },
           {
-            "key": "No",
+            "key": "not verified",
             "text": "No"
```
Comment (Rkareko) [inline opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json:488]: Same as https://github.com/OpenSRP/opensrp-client-reveal/pull/1290/files#r593193154

File: opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json (hunk 1 above)
Comment (lindagcaba) [reply opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json:484]: this was dones to help with saving the saving of values "verified" and "not verified" which is was they want to see on filled forms

File: opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json (hunk 2 above)
Comment (lindagcaba) [reply opensrp-reveal/src/main/assets/json.form/community_drug_distributor_supervisor_daily_summary_form.json:488]: same reason here

File: opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/EventRegisterFragmentPresenter.java [hunk 3]
```diff
@@ -92,6 +94,21 @@ private String mainSelect(String tableName, String mainCondition) {
     }
 
     protected String[] mainColumns(String tableName) {
+        if(Country.KENYA.equals(BuildConfig.BUILD_COUNTRY)){
```
Comment (Rkareko) [inline opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/EventRegisterFragmentPresenter.java:97]: This is code repetition. The only difference here is the new `DATA_COLLECTION_DATE`. Why not just optionally add it to the column array when the build country is KENYA?

Comment (lindagcaba) [reply opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/EventRegisterFragmentPresenter.java:97]: yes you're right,let me update that.

Comment (lindagcaba) [reply opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/EventRegisterFragmentPresenter.java]: I've addressed this.

**Review by Rkareko — APPROVED** [review]

**Pull request merged**